docs/source/guide/ml.md

```diff
@@ -68,8 +68,7 @@ Follow these steps to set up an example text classifier ML backend with Label St
    
 3. Start the ML backend server.
    ```bash
-   label-studio start my_ml_backend --init \ 
-     --ml-backends http://localhost:9090
+   label-studio-ml start my_ml_backend
```

Comment:
It would be nice to leave a note that you can add ml backend via command line

Reply:
I'll fix this in another PR